tests/fork/Makefile

```diff
@@ -27,7 +27,7 @@ tests:
 
 # no fork
 test-no-fork: rootfs
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) --app-config-path config-none.json rootfs /bin/fork nofork
+	TIMEOUT=360 $(RUNTEST) $(MYST_EXEC) $(OPTS) --app-config-path config-none.json rootfs /bin/fork nofork
```

Comment:
I'd love to understand why this simple test could take 6 minutes.

Reply:
Basically _something_ is causing the first test to get executed to take an undetermined amount of time. I have not been able to narrow this down more at this point, but my current thoughts are that one of the shared libraries the myst tool links again is doing something during the initialization of the library and that sometimes takes longer than it should.
It generally takes less than a couple of minutes, but sometimes it is a little more.
The hang happens _before_ main() is even hit in the process which is why I think this is something to do with the first test being called.

Reply:
I am not a fan of making tests pass by increasing the time outs or memory sizes. The actual question we should ask is: how could a previous version of Mystikos pass the test with a smaller time out or memory size. If we continue on the path of increasing time outs or memory sizes, we are on a deteriorating path.

Reply:
We should definitely rootcause this, but there is also the issue of this blocking PRs.